frontend/src/js/controllers/challengeCreate.js

```diff
@@ -81,7 +86,7 @@
     }
 })();
 
-/* This code can be used for creating challenge using UI feature.     
+/* This code can be used for creating challenge using UI feature.
```

Comment:
Please remove this change. I don't think it is a part of this PR.

Reply:
Haven't made any change here! Can't understand this part. It's same as the original code! Please explain.

Reply:
Your editor removed some space from the end. It should be removed but not in this PR.